samples/account-linking/csharp/Source/AccountLinking.Service/Services/KeyValueStore/DistributedCacheAdapter/KeyValueStoreDistributedCacheAdatper.cs

```diff
@@ -0,0 +1,49 @@
+using Microsoft.Extensions.Caching.Distributed;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Teams.Samples.AccountLinking.Service.KeyValueStorage;
+
+/// <summary>
+/// The KeyValueStoreDistributedCacheAdatper is a naive implementation of the <see cref="ITokenStore"> that stores the values into 
```

Comment:
This says it's an implementation of an ITokenStore interface, but it does not do that.